src/outputter.py

```diff
@@ -0,0 +1,29 @@
+def output_formatter(objects: list, serializer_function):
```

Comment:
Der skal ikke også laves tjek på "objects"? Eller det kan måske ikke lade sig gøre? Eller bruger vi bare anbefalingen?

Reply:
Jeg tænker ummidelbart at det ville være op til serializer funktionen. Da det vel er den der sætter restriktionerne for hvad den kan håndtere.